marklogic-data-hub/src/main/java/com/marklogic/hub/HubConfig.java

```diff
@@ -642,4 +642,8 @@
      * and then call this method.
      */
     void refreshProject();
+
+    String getMlUsername();
```

Comment:
These likely should always have been exposed, as multiple clients were casting HubConfig to HubConfigImpl to access these methods.